docs/_documentations/remote-configuring.md

```diff
@@ -0,0 +1,151 @@
+
+## Procedure
+
+To securely configure Codewind remote there are two options, configuring Kubernetes for Docker Desktop, or configuring a Cloud deployment that does not have an Ingress NGINX controller, for example, OpenShift. 
+
+### Configuring Kubernetes for Docker Desktop
+
+1. Validate you have an ingress controller installed on your cluster by typing the following command: [Mark to supply] 
+
+   If you are developing on Windows [get map local address from Mark]
+
+   If you do not have an ingress controller, install one using the following commands:
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml`
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud-generic.yaml`
+
+2. Create the remote deployment by entering the following `cwctl` command: 
+   `cwctl install remote \`
+     `--namespace {your_namespace}  \`
+     `--kadminuser admin \`
+     `--kadminpass <keycloakPassword>  \`
+     `--krealm codewind \`
+     `--kclient codewind  \`
+     `--kdevuser developer \`
+     `--kdevpass <userPassword>`
+
+### Configuring Cloud deployments that do not have an Ingress NGINX controller
+
+1. If you are deploying to a Cloud that does not have an Ingress NGINX controller, for example, OpenShift, you must provide the ingress value in the `cwctl install remote \` command during installation. Derive the `ingress` value from the cluster URL. [Needs more from Mark]
+2. Create the remote deployment by issuing the following example `cwctl` command: 
+
+  `cwctl install remote \`
+     `--namespace {your_namespace}  \`
+     `--kadminuser admin \`
+     `--kadminpass <keycloakPassword>  \`
+     `--krealm codewind \`
+     `--kclient codewind  \`
+     `--kdevuser developer \`
+     `--kdevpass <userPassword>`
+     `--ingress "apps.myopenshiftserver.10.20.30.40.nip.io`
+    
+### Codewind CLI command explanation and sample output
+
+The `cwctl install remote` command:
+1. Checks your Cloud is valid.
+2. Determines the ingress domain based on NGINX ingress that you added earlier. 
+3. Creates the SSL certificates for the keycloak server and adds these to the ingress endpoint, and then starts Keycloak. 
+4. It then populates all of the database entries inside Keycloak with the values that you added to the command. 
+5. Sets up the performance pods.
+6. Waits for Codewind to start. When Codewind has started, in Kubernetes, you see new four new pods including keycloak plus the services and the deployment configurations for those pods. If you are deploying on to OpenShift, you can use routes instead of ingress. 
+7. On successful completion of the command, the command returns a Gatekeeper URL which is the one and only secure remote access entry point for the service. 
+
+You see example output similar to the following, this for a deployment on Kubernetes:
+
+```
+~ cwctl --insecure install remote \
```

Comment:
Ensure to separately list a Windows cwctl invocation example or at least mention the cited one is for Linux/Mac only, since using "\" to separate the command line call into multiple lines won't work in the Windows command prompt

Reply:
Mentioned that the example is for `Linux/macOS`